src/logic/operate.js

```diff
@@ -0,0 +1,20 @@
+import Big from "big.js";
+
+const operate = (numberOne, numberTwo, operation) => {
+  let [x, y] = [numberOne, numberTwo];
```

Comment:
* [ ] You are creating an extra space `x` and `y`, it would be more efficient not to looking at your approach, if you were wrapping `numberOne` and `numberTwo` in big js and storing them in `x` and `y` to be used below then that would been a cleaner way, but since you are not doing that, it is better not to create any extra space because x and y are just copies of `numberOne` and `numberTwo`.

Reply:
This is a great suggestion. I have opted to do away with the assignments to `x` and `y` entirely. The first two parameters of the function are now named numOne and numTwo, and I refer to them directly in each `return` statement.